lib/bolt/bolt_option_parser.rb

```diff
@@ -1088,7 +1088,7 @@ def initialize(options)
       end
       define('--log-level LEVEL',
              "Set the log level for the console. Available options are",
-             "trace, debug, info, warn, error, fatal, any.") do |level|
+             "trace, debug, info, warn, error, fatal.") do |level|
```

Comment:
I think this also needs to be removed from `resources/bolt_bash_completion.sh`, `rakelib/pwsh.rake`, `lib/bolt_spec/plans.rb`, and `lib/bolt/config/options.rb`.

Reply:
I wish I knew why I included `any` in the first place... :|